turtlebot2_drivers/CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ if(BUILD_TESTING)
 endif()
 
 install(
-  FILES launch/turtlebot_carto_2d.py launch/turtlebot_carto_3d.py
+  FILES launch/turtlebot_carto_2d.py launch/turtlebot_carto_3d.py launch/turtlebot_follow.py
```

Comment:
can we now install launch directory rather than adding file one by one ? (as pointed out in https://github.com/ros2/turtlebot2_demo/pull/32#discussion_r121725059)